CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to AIConfig { }
+
+👋 Welcome and thank you for your interest in contributing to AIConfig! 
+
+---
+### **Ways to Contribute**
+
+1. **{ Code Contributions }**: 
+
+2. **{ Cookbooks }**: Showcase innovative ways and use cases to use AIConfig in your code through a cookbook.
+
+3.  **{ Bugs }**: File a bug report when you encounter issue and/or submit a fix for the bug.
+
+4. **{ Feature Requests }**: Submit feature or enhancements requests to help guide our roadmap.
+
+5. **{ Documentation }**: Contribute documentation to make it easier for others using through the library.
+
+---
+### **1. Code Contributions**
+
+```
+aiconfig/
+├─ cookbook/
+├─ cli/
+├─ editor/
+├─ aiconfig-docs/
+├─ python/
+│   ├── model-parsers/
+├─ typescript/
+│   ├── model-parsers/
+├─ extensions/
+```
+
+> *To Do: Add information on library structure and contribution opportunities*
```

Comment:
I think the most important section is missing, which is contributing model parsers